### PR TITLE
Add background color to the delay/error and wait screens.

### LINKED ISSUE #3393: [main site] error/wait/delay screen has transparent background
Affects train train-2013.05.08

screen shot 2013-05-14 at 14 13 33

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@jedp and @jrgm - could I get a review of this?

@jrgm - should this be merged into train-2013.05.08?

fixes #3393

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
The test failure is because #3391 has not been applied yet. I will rebase once that is fixed.

**Comment by jedp** [comment]
Testing now

File: resources/views/layout.ejs [hunk 1]
```diff
@@ -41,9 +41,9 @@
     </header>
 <% } %>
 
-    <div id="wait"><div class="contents"></div></div>
-    <div id="error"><div class="contents"></div></div>
-    <div id="delay"><div class="contents"></div></div>
+    <div id="wait" class="message_screen"><div class="contents"></div></div>
+    <div id="error" class="message_screen"><div class="contents"></div></div>
+    <div id="delay" class="message_screen"><div class="contents"></div></div>
```
Comment (ozten) [inline resources/views/layout.ejs:46]: Looks good, tested locally with Desktop Aurora.

**Comment by ozten** [comment]
r+

**Comment by jedp** [comment]
Looks great here too.  Merging this puppy.

**Pull request merged**